src/compiler/scala/tools/nsc/plugins/Plugin.scala

```diff
@@ -126,9 +127,9 @@ object Plugin {
 
     val fromLoaders = paths.map {path =>
       val loader = findPluginClassloader(path)
-      loader.getResource(PluginXML) match {
-        case null => Failure(new MissingPluginException(path))
-        case url =>
+      loader.getResources(PluginXML).asScala.toList.lastOption match {
```

Comment:
This is a second line of defence, in case a `Global` subclass overrides `findPluginClassloader` and doesn't special case parent delegation for `plugin.xml` as I've done in `Plugins.scala`.